user-service/internal/models.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import "github.com/go-redis/redis/v8"
+
+func GetMoney(username string) int {
+
+	val, err := Rdb.Get(Ctx, username).Int()
+
+	if err == redis.Nil {
+		return -1
+	} else if err != nil {
+		panic(err)
+	}
+		
+	return val
+}
+
+func AddUser(username string, money string)  {
+
+	err := Rdb.Set(Ctx, username, money,0).Err()
+	if err != nil {
+		panic(err)
+	}
+
+}
+
+func DecreseMoney(username string, money int) {
+
+	val1, err1 := Rdb.Get(Ctx, username).Int()

```

Comment:
There aren't any other variables named `val` which means there's no reason for this one to have the name `val1`. 
`moneyAmount` or something similar probably makes more sense for the name, because the `val` is kind of generic